INSTALL.rst

```diff
@@ -55,9 +55,79 @@ Finally run the tests to make sure all is setup correctly:
     python setup.py test
 
 
+Installation for developers with Docker
+---------------------------------------
+
+Grab the code from github, you can optionally follow the above steps to create the
+virtual environment, but that is not neccessary – might be helpful if your IDE wants
+to install dependencies for suggestions though.
+
+Then install the test dependencies inside Docker:
+
+.. code-block:: console
+
+    docker-compose -f docker-compose.deps.yml run --rm pip
+
+To run the tests (e.g. unit):
+
+.. code-block:: console
+
+    docker-compose -f docker-compose.test.yml run --rm unit
+
+
+Installation for testing with inspire-next in Docker
+----------------------------------------------------
+
+Follow the directions to create the virtual environment as above and grab the code from github.
+
+In ``inspire-next`` in the ``services.yml`` file add a new volume in the static section, it should
+look like so now:
+
+.. code-block:: yaml
+
+    static:
+      image: busybox
+      volumes:
+        - "/local/path/to/hepcrawl/repo:/hepcrawl_code"  # <- added
+        - ".:/code"
+        - "${DOCKER_DATA}/tmp/virtualenv:/virtualenv"
+
+
+Then in the ``docker-compose.deps.yml`` we need to tell ``scrapyd-deploy`` service to work with
+the new code by changing the ``working_dir`` to ``/hepcrawl_code/hepcrawl``:
+
+.. code-block:: yaml
+
+    scrapyd-deploy:
+      extends:
+        file: services.yml
+        service: base
+      # working_dir: /virtualenv/lib/python2.7/site-packages/hepcrawl
+      working_dir: /hepcrawl_code/hepcrawl
+      command: scrapyd-deploy
+      volumes_from:
+        - static
+      links:
+        - scrapyd
+
+Last step is to deploy the spiders to `scrapyd` (see more in :doc:`Deployment <operations>`):
```

Comment:
This operations sections looks outdated, and it's not the inspire-next one, probably you might want to point to https://inspirehep.readthedocs.io/en/latest/harvesting.html, and change the page there (as it's outdated too :/).